sapi/phpdbg/phpdbg.c

```diff
@@ -912,21 +912,21 @@ void phpdbg_register_file_handles(void) /* {{{ */
 
 	ic.value = zin;
 	Z_CONSTANT_FLAGS(ic.value) = 0;
-	ic.name = zend_string_init(ZEND_STRL("STDIN"), 0);
```

Comment:
These don't need to be interned. Instead you should just call `zend_string_release()` after registering the constant.